x/concentrated-liquidity/grpc_query.go

```diff
@@ -102,3 +102,30 @@ func (q Querier) Params(goCtx context.Context, req *types.QueryParamsRequest) (*
 	ctx := sdk.UnwrapSDKContext(goCtx)
 	return &types.QueryParamsResponse{Params: q.Keeper.GetParams(ctx)}, nil
 }
+
+// LiquidityDepthsForRange returns liquidity depths for the given range.
+func (q Querier) LiquidityDepthsForRange(goCtx context.Context, req *types.QueryLiquidityDepthsForRangeRequest) (*types.QueryLiquidityDepthsForRangeResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	maxRange := sdk.NewInt(10000)
+
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	if req.LowerTick.GT(req.UpperTick) {
+		return nil, status.Error(codes.InvalidArgument, "lower tick is greater than upper tick")
+	}
```

Comment:
We might have an error status already premade in errors.go (that also returns what the lower and upper tick that was provided for clarity)

Reply:
Changed to the error suggested!